app/code/Magento/MediaGalleryApi/Api/IsPathBlacklistedInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
```

Comment:
Look how it uses in media gallery UI module

Reply:
looked